utils/utils.go

```diff
@@ -31,3 +31,10 @@ func ValidateWhitelistedAddress(address string) error {
 	}
 	return nil
 }
+
+func Min(a int, b int) int {
```

Comment:
seems like this is unused. if so, let's rm?

Reply:
https://github.com/Stride-Labs/stride/pull/70#discussion_r917017476

It's used in the claim function

Reply:
resolved above